pkg/asset/agent/manifests/clusterdeployment_test.go

```diff
@@ -25,8 +27,34 @@ func TestClusterDeployment_Generate(t *testing.T) {
 	}{
 		{
 			name:          "missing-config",
+			dependencies:  unittest.GetEmptyTestAssets(),
 			expectedError: "missing configuration or manifest file",
 		},
+		{
+			name:         "default",
+			dependencies: unittest.GetTestAssetsWithValidInstallConfig(),
```

Comment:
The idea here is to read the valid/invalid inputs for install config from a single place as that will be needed by other asset unit tests when we enable them.